src/Okta.DNX.OAuth.ResourceServer/Models/TodoRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Okta.DNX.OAuth.ResourceServer.Models
+{
+    public class TodoRepository : ITodoRepository
+    {
+        private static ConcurrentDictionary<string, TodoItem> _todos =
```

Comment:
For the record, this controller is a copy/paste for a Microsoft tutorial (https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api) so I'm going to leave it as is.